packages/core/src/browser/style/variables-dark.useable.css

```diff
@@ -148,13 +148,14 @@ is not optimized for dense, information rich UIs.
   --theia-removed-color0: rgba(230, 0, 0, 0.8);
   --theia-modified-color0: rgba(0, 100, 150, 0.8);
 
-  --theia-highlight-background-color: var(--md-purple-A400);
-  --theia-highlight-color: var(--theia-content-font-color0);
-
+  
   /* Colors to highlight words in widgets like tree or editors */
-
+  
   --theia-word-highlight-color0: rgba(81, 92, 106, 0.7);
   --theia-word-highlight-color1: rgba(255, 255, 255, 0.04);
+  --theia-word-highlight-match-color0: rgba(234, 92, 0, 0.33);
+  --theia-word-highlight-match-color1: rgba(234, 92, 0, 0.5);
+  --theia-word-highlight-replace-color0: rgba(155, 185, 85, 0.2);
```

Comment:
`match` and `replace` is actually still a bit special, don't you think? We could have `--theia-word-highlight-color0` to `--theia-word-highlight-color5`. Disadvantage with that would be that there isn't a real order of colours like in `--theia-info-color...` from darker to lighter colour. 
What do you think? Leave it as it is?

Reply:
I am a bit puzzled, I thought you [agreed with my proposed change](https://github.com/theia-ide/theia/pull/2037#discussion_r192995400). To answer your question; I do not feel `match` and `replace` more specific than `added`, `removed`, and `modified` but if you have a better idea, let me know and I do another adjustment.

Reply:
Yeah, I am sorry for that. I was just overthinking it and made it more complicated as it actually is...lets leave it. :-)